### PR TITLE
Instagram sign in

### PR DESCRIPTION
**Pull request by kluangkote**
Part of #1 
Frontend implementation of IG sign in process

### CODE REVIEW

**Comment by kluangkote** [comment]
By the way, this is rebased on Yun's branch

File: src/components/auth/Login.js [hunk 1]
```diff
@@ -58,12 +63,18 @@ class Login extends Component {
       });
   }
 
+  redirectToIG = event => {
+    let client_id = 'f296ed176092447582392cbec8f2d914'
+    let redirect_uri = 'http://localhost:3000/influencer/create_account'
```
Comment (xiaoxuwu) [inline src/components/auth/Login.js]: move these to .env

Comment (kluangkote) [reply src/components/auth/Login.js]: thanks. wasn't sure where to put it

Comment (xiaoxuwu) [reply src/components/auth/Login.js]: look at AxiosClient for how to use .env variables. Make sure the variables begin with REACT_APP_

Comment (xiaoxuwu) [reply src/components/auth/Login.js]: and sorry if I wasn't clear but client_id should be moved to .env, redirect_uri should be constructed from some base_url_frontend in .env and the relative path /influencer/create_account. Hope this makes sense.

**Comment by kluangkote** [comment]
> LGTM!

Cool. This is based on Yun's PR, so I'll wait for that to merge first.

**Pull request merged**